common/python/schema_validation/data/sdk_WorkOrderSubmit.json

```diff
@@ -0,0 +1,87 @@
+{
```

Comment:
Why we need this since we have WorkOrderSubmit.json. Can't we load WorkOrderSubmit.json and validate the required subset.

Reply:
sdk_WorkOrderSubmit.json has few additional check and also fewer parameters than WorkerOrderSubit request.

Reply:
To be consistent with TC spec, I would suggest not to have a separate WorkOrderSubmit and sdk_WorkOrderSubmit.  But then this would mean we need to add inData, outData as part of work order params.

Reply:
Sdk forms the request step by step process. Few Params like encryption session key are being generated by the sdk itself. There is a separate json schema for sdk because the request is not ready to be posted and cannot do a  complete validation like WorkerLook up. That would required changes to generic client itself.  If the user opts to add Indata in that case Indata gets validated before getting added to the request.

Reply:
Please raise an issue to remove the sdk_WorkOrderSubmit.json later to make Avalon clients to generate encryptedSessionKey outside of SDK

Reply:
done